src/libs/dates.ts

```diff
@@ -10,11 +10,6 @@ import * as portal from './portal';
  * @returns A Moment.js object with the start of school.
  */
 export function thirdWednesdayAugust(year: number) {
-	const current = moment();
-	if (typeof year !== 'number' || year % 1 !== 0) {
```

Comment:
This checks to make sure that year isn't a decimal, don't know if it's important

Reply:
I figured it wasn't because `moment` automatically floors the year that's passed in.